lib/client/src/client.rs

```diff
@@ -1162,6 +1162,17 @@ impl MangoClient {
         let account = self.mango_account().await?;
         let open_orders = account.serum3_orders(market_index).unwrap().open_orders;
 
+        let ix = self.serum3_settle_funds_instruction(s3, base, quote, open_orders);
+        self.send_and_confirm_owner_tx(vec![ix]).await
+    }
+
+    pub fn serum3_settle_funds_instruction(
+        &self,
+        s3: &Serum3MarketContext,
+        base: &TokenContext,
+        quote: &TokenContext,
+        open_orders: Pubkey,
+    ) -> Instruction {
```

Comment:
this should return a PreparedInstruction. Could start with cu_per_mango_instruction (even though settlement is probably cheaper than that) (could also be a separate PR)